src/content/get-started/fundamentals/index.md

```diff
@@ -18,67 +19,67 @@ you how to _program_, it's about teaching you
 how Flutter works.
 
 :::note
-This first week experience is still a work
+These fundamentals docs are still a work
 in progress and we welcome your feedback!
 Please consider filling out the survey
 listed at the bottom of this page and on the
 new subject pages in this section.
 :::
 
-As you might know, Flutter uses the [Dart language][].
-If you have experience with other object-oriented
-languages, like Java, C++, or Swift,
-Dart should feel familiar to you.
-As of this writing,
-[Dart is one of the fastest growing languages][dart-lang],
-in part, thanks to Flutter.
-
-[Dart language]: {{site.dart-site}}
-[dart-lang]: https://twitter.com/MiSvTh/status/1732002450641400276?cxt
-[your first Flutter codelab]: {{site.codelabs}}/codelabs/flutter-codelab-first
-
 We suggest that you work through the
 following subjects in the listed order.
 
-1. [Widget fundamentals][]
+1. [Intro to Dart][] (Optional)
+: As you might know, Flutter uses the [Dart language][].
+   If you have experience with other object-oriented
+   languages, like Java, C++, or Swift,
+   Dart should feel familiar to you.
+   As of this writing,
+   [Dart is one of the fastest growing languages][dart-lang],
+   in part, thanks to Flutter.
+2. [Widget fundamentals][]
 : Learn about one of the primary building blocks
   of a Flutter application, `Widgets`. 
```

Comment:
Since `Widgets` is not a type, either drop the code font or adjust the sentence.

```suggestion
  of a Flutter application, widgets.
```